### ISSUE TITLE
Missing some CanvasRenderingContext2D API?

### ISSUE THREAD

**Issue by clkbug** [fable-compiler/fable-browser#45]
Hello,

It looks like some  CanvasRenderingContext2D API are missing in [Browser.Dom.fs](https://github.com/fable-compiler/fable-browser/blob/master/src/Dom/Browser.Dom.fs#L2658) .

```
    abstract drawFocusIfNeeded: HTMLElement -> unit
    abstract ellipse: x: float * y: float * radiusX: float * radiusY: float * rotation: float * startAngle: float * endAngle:float * ?anticlockwise: bool -> unit
    abstract isPointInStroke: x:float * y: float -> bool
```

https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/ellipse

**Comment by MangelMaxime** [comment]
It should be fixed in version 2.1.0

**Comment by clkbug** [comment]
Thank you for your quick response.
